ui/apps/platform/src/services/ReportsService.ts

```diff
@@ -119,8 +119,36 @@ export function fetchReportHistory({
         });
 }
 
+export type FetchOnDemandReportHistoryServiceProps = {
```

Comment:
Naming nit: ending with `Props` makes me think these options belong to a react component.

Reply:
Oops, I copied from `fetchReportHistory` when making this new function. That one also uses `Props`. I didn't catch that. `Args` might make more sense here.